### PR TITLE
added javascript validation hw

### PR DESCRIPTION
**Pull request by dinhngoc87**

### CODE REVIEW

**Comment by dinhngoc87** [comment]
user1 = {
  first_name: "Truo$ng",
  last_name: "Dung",
  email: "dung@test.com",
  password: "Dung123",
  password_confirmation: "Dung123",
};
image

File: dinhngoc87/javascript-validation/server.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+app.get("/", (req, res) => {
+  res.send("Hello World!");
+});
+
+app.post("/api/users/register", (req, res) => {
+  const { error, value } = schema.validate(req.body);
+  if (error) {
+    //   res.status(400);
+    //   res.json("123");
+    res.status(400).send(error.toString());
```
Comment (revskill10) [inline dinhngoc87/javascript-validation/server.js:32]: Can throw ValidationError here.

Comment (revskill10) [reply dinhngoc87/javascript-validation/server.js:32]: We already have centralized ErrorHandling, so controller code can just throw error.

Comment (dinhngoc87) [reply dinhngoc87/javascript-validation/server.js:32]: hmm If I throw an Error(error.toString()), then it throws an error on server side.

**Pull request merged**